### ISSUE TITLE
About the loss

### ISSUE THREAD

**Issue by yanmengdi123** [wentaozhu/AnatomyNet-for-anatomical-segmentation#24]
Hello, I want to use the mixed loss function(focal and dice loss) you mentioned in the paper. I don't know which part of the code I should look at, but what I downloaded is "baselineDiceFocalLoss.py".Do you want to make sure that this part of the code I'm looking at is correct? Thank you very much!
您好，我想借鉴您在论文中提到的混合损失函数应用到我的篡改检测模型中，准备参考baselineDiceFocalLoss.py，不知道看的代码是不是对应的？非常感谢您！

**Comment by wentaozhu** [comment]
https://github.com/wentaozhu/AnatomyNet-for-anatomical-segmentation/blob/master/src/baselineDiceCrossEntropy.py#L551           Replace cross entropy with focal loss in https://github.com/wentaozhu/AnatomyNet-for-anatomical-segmentation/blob/master/src/AnatomyNet.py#L528 Thank you!

**Comment by yanmengdi123** [comment]
> https://github.com/wentaozhu/AnatomyNet-for-anatomical-segmentation/blob/master/src/baselineDiceCrossEntropy.py#L551 Replace cross entropy with focal loss in https://github.com/wentaozhu/AnatomyNet-for-anatomical-segmentation/blob/master/src/AnatomyNet.py#L528 Thank you!

Hi, thank you very much for your reply. What you mean above is to refer to the method of using the mixed loss function in the code of Baselineditecrossentropy, and then make the mixed loss function of DICE and Focal Loss in your paper. Is my understanding correct?  In addition, the following link "anatiomynet. py#L528" is referring to your mixed loss function code from line 528?
 Thank you very much!
首先非常感谢您的回复。您上面的意思是参考baselineDiceCrossEntropy的代码里使用混合损失函数的方法，然后制作的您论文中dice and focal loss 的混合损失函数，我的理解是正确的吗。另外，您给的下面那个链接“AnatiomyNet.py#L528"的部分，是说让我从528行参考您的混合损失函数代码是吗？ 
非常感谢您！

**Comment by wentaozhu** [comment]
Yes.
